en_US/configuration/logs.md

```diff
@@ -1,27 +1,16 @@
 # Logs
 
-Logs provide a reliable source of information for troubleshooting and system performance optimization. You can find the record about the access, operating, or network issues from EMQX logs.
+This page introduces how to configure logging behaviour for EMQX. You can configure EMQX logs with Dashboard or configuration files. To configure with EMQX Dashboard, you can click **Management** -> **Log** on the left navigation menu to configure. For more detailed descriptions of logs and Dashboard configurations, see [Logs and observability - Logs](../observability/log.md).
```

Comment:
I changed it back to **Management** -> **Log**, because in 5.4, the Dashboard UI is "Log", not "Logging". We can update it after the PR for log format updating is submitted to the release-5.6 branch.